DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -138,30 +142,39 @@ public IActionResult Summary()
         [HttpPost]
         public IActionResult Summary(SummaryViewModel model)
         {
+            var data = TempData.Peek<DelegateRegistrationData>()!;
             if (!ModelState.IsValid)
             {
-                var data = TempData.Peek<DelegateRegistrationData>();
-                var viewModel = MapToSummary(data!);
+                var centre = centresDataService.GetCentreName((int)data.LearnerInformationViewModel.Centre!);
+                var jobGroup = jobGroupsDataService.GetJobGroupName((int)data.LearnerInformationViewModel.JobGroup!);
+                var viewModel = MappingHelper.MapToSummary(data, centre!, jobGroup!);
                 viewModel.Terms = model.Terms;
                 return View(viewModel);
             }
 
-            // TODO HEEDLS-396 create new user and redirect to confirmation page
-            return View(model);
+            var baseUrl = ConfigHelper.GetAppConfig()["CurrentSystemBaseUrl"];
+            var candidateNumber = registrationService.RegisterDelegate(MappingHelper.MapToDelegateRegistrationModel(data), baseUrl);
+            if (candidateNumber == "-1")
+            {
+                Response.StatusCode = 500;
+                return View("../LearningSolutions/Error/UnknownError");

```

Comment:
Could we redirect to the LearningSolutions/Error page instead of this (it has 500 error by default)